src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -349,7 +349,8 @@ GOOGLE_OAUTH_CONFIG = {
         "auth_uri": "https://accounts.google.com/o/oauth2/auth",
         "token_uri": "https://oauth2.googleapis.com/token",
         "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
-        "client_secret": '{{ GOOGLE_OAUTH_CLIENT_SECRET }}'
+        "client_secret": '{{ GOOGLE_OAUTH_CLIENT_SECRET }}',
+        "redirect_uris": ["https://staging.commcarehq.org/oauth/google_sheets_oauth/callback/"],
```

Comment:
Just curious why this needs to be a list? Is it a config constraint or is there a plan to add more uris to this in the future?

Reply:
I will need to add the prod uri once we push gsheets to prod, and then probably like india and swiss if gsheets is going to be used on those environments.

Reply:
is there any way to add this outside of the localsettings config here? It would be nice to build this url from referencing the actual view. I feel like this URL does not need to live here like this